chatbot/src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration scan="true">
+
+	<appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<charset>UTF-8</charset>
+			<Pattern>%d{yyyy-MM-dd HH:mm:ss.SSS} %-5level [%thread] "chatbot_ms" [%C] %logger{36} - %msg%n</Pattern>
+		</encoder>
+	</appender>
+
+	<appender name="FILE"
+		class="ch.qos.logback.core.rolling.RollingFileAppender">
+		<file>C:/logs/chatbot.log</file>
```

Comment:
Windows path used in two places in configuration of chatbot log file. Is this correct?

Reply:
To run ELK in docker a shared volume between them is needed that it is a windows path for now.